File: climb-log-flutter/lib/climasys/features/account_view/models/edit_account_details_request.dart
// models/route.dart
class EditAccountDetailsRequest {
  final String friendlyName;
  final String? bioText;

  EditAccountDetailsRequest({
    required this.friendlyName,
    required this.bioText
  });

  Map<String, dynamic> toJson() => {
    'friendlyName': friendlyName,
    'bioText': bioText
  };
}

File: ClimbLog.Api/ClimbLog.Application/Features/Login/Commands/EditAccountDetails/EditAccountDetailsRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClimbLog.Application.Features.Login.Commands.EditAccountDetails;
public class EditAccountDetailsRequest
{
    public required string FriendlyName { get; set; }
    public string? BioText { get; set; }
}
